nix/package.nix

```diff
@@ -12,20 +14,27 @@ stdenv.mkDerivation rec {
   src = ./..;
 
   nativeBuildInputs = [ zig scdoc pkg-config ];
-
-  buildInputs = [
-    libxml2
-  ];
+  buildInputs = [ libxml2 ];
 
   dontConfigure = true;
 
   preBuild = ''
     export HOME=$TMPDIR
+    cp -r ${zig-libxml2-src} ./vendor/zig-libxml2
```

Comment:
We make the zig-libxml2 repo available this way since `src = ./..;` pulls from the pure flake and that doesn't include submodules